src/Codeception/Module/Parameter.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Codeception\Module;
+
+use Codeception\Module as CodeceptionModule;
+
+/**
+ * Class Parameter
+ */
+class Parameter extends CodeceptionModule
```

Comment:
please don't... this doesn't look like a real module.
I think parameters should be taken in using Scenario->current

```
$scenario->current('paramName')
```